test/full_node.js

```diff
@@ -0,0 +1,50 @@
+var Trust = require('../');
+var bcoin = require('bcoin');
+var testHelpers = require('./helpers');
+var consensus = require('bcoin/lib/protocol/consensus');
+var sinon = require('sinon');
+var should = require('should');
+require('should-sinon');
+
+describe('FullNode', () => {
+  var node = null;
+  var walletDB = null;
+  sinon.spy(Trust.TrustIsRisk.prototype, 'addTX');
+
+  beforeEach(() => testHelpers.getNode().then((n) => {
+    node = n;
+  }));
+
+  beforeEach(() => testHelpers.getWalletDB(node).then((w) => {
+    walletDB = w;
+  }));
+
+  afterEach(() => walletDB.close());
+  afterEach(() => node.close());
+
+  it('should be a bcoin instance', () => {
+    should(node).be.an.instanceof(bcoin.fullnode);
+  });
+
+  it('should call trust.addTX() on every transaction', async function() {
+    var sender = await testHelpers.getWallet(walletDB, 'sender');
+    var receiver = await testHelpers.getWallet(walletDB, 'receiver');
```

Comment:
That parameter is the ID parameter of the wallet and needs to be unique. Here I'm setting it to something descriptive.